pages/api/v1/schedule/index.ts

```diff
@@ -8,13 +8,27 @@ const cors = Cors({
   methods: ['GET', 'HEAD'],
 });
 
+type SeasonType = "Pre-Season" | "Regular Season" | "Playoffs";
+
+export interface Game {
+  season: string;
+  homeTeam: string;
+  homeScore: number;
+  awayTeam: string;
+  awayScore: number;
+  overtime: number;
+  shootout: number;
+  played: number;
+  type: SeasonType
+}
+
 export default async (
   req: NextApiRequest,
   res: NextApiResponse
 ): Promise<void> => {
   await use(req, res, cors);
 
-  const { league = 0, season: seasonid } = req.query;
+  const { league = 0, season: seasonid, type} = req.query;
```

Comment:
can we make the type default here as well? like do `type = "Regular Season"`

Reply:
I'll make it happen